core-bundle/tests/Routing/ResponseContext/CoreResponseContextFactoryTest.php

```diff
@@ -124,7 +124,7 @@ public function testContaoWebpageResponseContext(): void
         $pageModel->title = 'My title';
         $pageModel->description = 'My description';
         $pageModel->robots = 'noindex,nofollow';
-        $pageModel->enableCanonical = '1';
+        $pageModel->enableCanonical = 1;
```

Comment:
My thinking was that we should simulate in the tests how the value would be in a real application. Under PHP 8.1 with PDO, the value will be `1`, not `true`. Except when #4800 gets merged, then it would be `true`.